ZENKO-987 refactor: import external charts

giacope opened the pull request:
No need to review the files in detail since PR is only importing charts that we are already using to this repo.

bert-e commented:
# Hello giacomoguiulfo,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get
information on this process.





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:



* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @ssalaues

* @vrancurel

* @giacomoguiulfo

* @NicolasT

giacomoguiulfo commented:
@bert-e wait

giacope committed:
ZENKO-987 feat: import upstream grafana chart

Chart v1.11.6
commit d3f70f51323c668f2cdf02b7b61969fccc1de4dc

giacope committed:
ZENKO-987 feat: import upstream kafka chart

Chart v0.7.1
commit e11cb70205edf12d581e213a567c66e4a8b2a1ea

giacope committed:
ZENKO-987 feat: import upstream zookeeper chart

Chart v1.0.0
commit ab579ad0cf86743cb08510f44ffe43ddd2f9d4e1

giacope committed:
ZENKO-987 chore: rename kafka and zookeeper

The following directories have been renamed:

kafka -> zenko-queue
zookeeper -> zenko-quorum

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/064d49333913847e51af7eba4fffb0e28ac81854) did not succeed in branch improvement/ZENKO-987-zenko-charts-refactor.

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/0.9`




There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of Release Engineering will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of Release Engineering now.

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/0.9`




Please check the status of the associated issue ZENKO-987.

Goodbye giacomoguiulfo.
